proto/daemon/v1alpha/sites.proto

```diff
@@ -0,0 +1,56 @@
+syntax = "proto3";
+
+package com.mintter.daemon.v1alpha;
+
+import "site/v1alpha/site.proto";
+
+option go_package = "mintter/backend/genproto/daemon/v1alpha;daemon";
+
+// Sites API allows to configure the remote Mintter Sites inside the local app.
+// To interact with the existing site, the local daemon will expose the Site API
+// interface as well (see /proto/site/v1alpha/site.proto), requiring a gRPC metadata
+// header with the hostname of the actual site to know the endpoint where to proxy the request.
+service Sites {
```

Comment:
This is the local API. It only needs a way to add and list sites. Well, maybe remove also. The rest is exposed by the actual Site and proxied by the local daemon with a special header.

Need to define the name for the header, so that the frontend could specify the actual hostname to talk to.

Reply:
What would be the daemon's behavior about that header? 
If it sees that metadata header then it calls the method remotely (on the provided host in the header) and if it does not have that metadata, then does the call locally? @burdiyan